### ISSUE TITLE
Comments on Abstract

### ISSUE THREAD

**Issue by MPillas** [MPillas/S240422ed#3]

**Comment by tpradier** [comment]
- via their gravitational-wave emission -> define here GW, used in "public GW information"
- replace mdyn,mwind  by something like dynamical ejecta and post-merger disk wind ejecta
- define theta (theta < 25°)

**Comment by alitakey** [comment]
The title does not include any information about which GW run (O4 or O4a).

**Comment by alitakey** [comment]
2nd line: kilonovae (KNe) or kilonova(KN)

**Comment by alitakey** [comment]
The abstract does not include information about the results and conclusion about the other two events GW230529 and S230627c.

**Comment by alitakey** [comment]
Is there result from comparison of O4 run and previous tuns?

**Comment by alitakey** [comment]
The abstract does not include sentences on the aims and conclusion of the paper.

**Comment by santier14** [comment]
@alitakey yes we did not mention O4a, or b as it is O4a + a bit of b :p and "
The abstract does not include information about the results and conclusion about the other two events GW230529 and S230627c." --> yes, we will include it :)

**Comment by santier14** [comment]
@alitakey "no objectif" ? ohhh :p , it is written "In this work, we focus on NSBH merger populations
and compare the optical data collected by the commu-
nity on S230518h, GW230529, S230627c and S240422ed.
For each event, different levels of GW information are
public, from raw data for published events to informa-
tion contained in the publicly distributed LVK alerts.
The method we develop in this article consists of test-
ing consistencies between the EM and GW data. " and "n particular, we propose an in-
dicator to evaluate the possibility of the presence of a
KN in the optical observations triggered from the GW
events."
